lib/engine/g_1822/company.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require_relative '../company'
+
+module Engine
+  module G1822
+    class Company < Company
+      attr_reader :type, :removed, :header_name, :header_color, :header_text_color
+
+      def initialize(game, **opts)
+        @type = opts[:type]&.to_sym || :private
+        @header_name = opts[:header_name] || 'PRIVATE COMPANY'
+        @header_color = game.registered_color(opts[:header_color]) || :yellow
+        @header_text_color = opts[:header_text_color] || :black
+        @removed = false
```

Comment:
Do you need to track removed or simply remove in the game class when the action happens? `@companies.delete(company)`

Reply:
It was a beginning of a thought. I do remove it, therefor i can remove the remove attribute.

Reply:
remove this class

Reply:
Hmm i still need the name for the company. If we gonna remove the coloring name is still needed 'PRIVATE COMPANY' in the view.

Reply:
Ok will remove it.